package.json

```diff
@@ -4,17 +4,31 @@
   "name": "bem-components",
   "version": "3.0.0",
   "description": "BEM Components Library",
+  "homepage": "https://bem.info/libs/bem-components/",
   "repository": {
     "type": "git",
     "url": "git://github.com/bem/bem-components.git"
   },
+  "bugs": "https://github.com/bem/bem-components/issues",
   "keywords": [
-    "bem"
+    "bem",
+    "components",
+    "controls",
+    "ui"
   ],
   "engines": {
     "node": ">=4"
   },
+  "files": [
+    "common.blocks",
+    "desktop.blocks",
+    "touch.blocks",
+    "design"
+  ],
```

Comment:
Would you mind to move tests, gemini-s, etc out of the package with `.npmignore`?